Unite/Unite/Controllers/ChatController.cs

```diff
@@ -29,4 +29,11 @@ public ChatController(AssistantService assistantService)
         
         return Ok(response);
     }
+
+    [HttpGet("health")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    public IActionResult HealthCheck()
+    {
+        return Ok();

```

Comment:
The endpoint path in the Jira ticket is mentioned as 'checkhealthsystem', but the implemented path in the PR is 'health'. Could you please update the PR to match the path specified in the Jira ticket?